### PR TITLE
issue #49: fixed the arguement names for control_go_to

### LINKED ISSUE #49: `control_go_to` uses the wrong JSON arguments
`control_go_to` currently accepts `duration` and `velocity` but the API excepts `time` and `max_speed` (https://automata.tech/docs/v/3.1.0/api/controls/#go-to).

While `duration` renamed `time` in the request body, `velocity` is kept as-is, triggering a 400.

We should rename `velocity` into `max_speed` completely (as it's broken anyway) and add `time` as well as `duration` to keep the function backward compatible.

### PR DESCRIPTION
**Pull request by Charlesworth**
#49 

Fixed the arg names for control_go_to to match the current API: https://automata.tech/docs/v/3.1.0/api/controls/#go-to

### CODE REVIEW

**Comment by Charlesworth** [comment]
added discussion point to the issue

File: evasdk/eva_http_client.py [hunk 1]
```diff
@@ -328,12 +328,13 @@ def control_run(self, loop=1, wait_for_ready=True, mode='teach'):
             time.sleep(0.1)     # sleep for small period to avoid race condition between updating cache and reading state
             self.control_wait_for(RobotState.READY)
 
-    def control_go_to(self, joints, wait_for_ready=True, velocity=None, duration=None, mode='teach'):
+
```
Comment (LouisBrunner) [inline evasdk/eva_http_client.py:331]: Extra line, IDK if the linting will complain?

Comment (Charlesworth) [reply evasdk/eva_http_client.py:331]: I added the extra line to make it the same as the other methods, linter doesn't seem to mind. Not sure where I got that double line between each method from but I saw it somewhere, maybe an old PIP?

**Comment by Charlesworth** [comment]
Should be good for a second look over, had some issues using `time` as the arguement name as its accidently overwrote the time lib that is used throughout. In parlance with some of our other arguments I changed it to time_sec, I figure this should still be readable to users and means we will not have to alias the time library which is used liberally throughout (lots of sleeps unfortunately!). There is a good arguement to be made here for just keeping it as `duration` at this point

File: evasdk/eva_http_client.py (hunk 1 above)
Comment (LouisBrunner) [reply evasdk/eva_http_client.py:331]: No idea, as long as the linter is happy 🤷

**Review by LouisBrunner — DISMISSED** [review]

**Review by LouisBrunner — APPROVED** [review]

**Pull request merged**